.github/workflows/tests.yml

```diff
@@ -126,6 +126,40 @@ jobs:
           s3cmd --host localhost --host-bucket="localhost/%(bucket)" --access_key=fooAccessKey --secret_key=fooSecretKey --no-ssl put -P ~/test.txt s3://testbucket
           curl -s http://localhost/testbucket/test.txt | grep -F hello-radosgw
 
+      - name: Test Cluster Config
+        run: |
+          set -eux
+          cip=$(ip -4 -j route | jq -r '.[] | select(.dst | contains("default")) | .prefsrc' | tr -d '[:space:]')
+          
+          # pre config set timestamp for service age
+          ts=$(sudo systemctl show snap.microceph.osd.service -f json-pretty | grep -m1 ActiveEnterTimestampMonotonic | awk '{n=split($1,A,"="); print A[n]}')
```

Comment:
The `systemctl show` command doesn't do json. Also you can directly ask for a property. Suggest to use something like:
```
sudo systemctl show --property ActiveEnterTimestampMonotonic snap.microceph.osd.service | cut -d= -f2
```

Reply:
ack